src/nidm/experiment/Query.py

```diff
@@ -1,33 +1,5 @@
-# *****************************************************************************
-# *****************************************************************************
-#  nidm_query.py
-#  License: GPL
```

Comment:
@dbkeator Please confirm that it is Ok to assume that this file is also under Apache 2 license as the rest of the codebase, or it has to be GPL?